packages/insured-bridge-relayer/src/RelayerHelpers.ts

```diff
@@ -35,34 +35,42 @@ export async function approveL1Tokens(
   }
 }
 
-// Iterate over provided list of whitelisted L1 tokens and remove any that are not whitelisted on a specific L2.
+// Iterate over queried list of whitelisted L1 tokens and remove any that are not whitelisted on a specific L2.
 // The Relayer will use this whitelist to determine which deposits to relay. By default, the whitelist will be set
-// to the list of all possible tokens provided in the RateModels dictionary. However, not all L1 tokens in this
+// to the list of all possible tokens provided in the on-chain RateModels dictionary. However, not all L1 tokens in this
 // dictionary will be whitelisted for all L2 deposit boxes.
 export async function pruneWhitelistedL1Tokens(
   logger: winston.Logger,
   l1Client: InsuredBridgeL1Client,
-  l2Client: InsuredBridgeL2Client,
-  whitelistedRelayL1Tokens: string[]
+  l2Client: InsuredBridgeL2Client
 ): Promise<string[]> {
   await Promise.all([l2Client.update(), l1Client.update()]);
```

Comment:
unrelated to this PR but it's not super ideal that we do the first client updates here :/

Reply:
should we update the clients before calling `pruneWhitelistedL1Tokens` and removing it from this method?

Reply:
Thoughts? https://github.com/UMAprotocol/protocol/pull/3673/commits/b2225df1436ad895ddaf7525bb0067f5b8461141

Reply:
ye that makes sence! thanks for making the change.